packages/botonic-plugin-flow-builder/tests/keyword.test.ts

```diff
@@ -12,14 +12,25 @@ describe('Check the contents returned by the plugin using keywords', () => {
   test.each(['reset', 'hola', 'HOLA'])(
     'The initial content is displayed when the user sends the %s text',
     async (inputData: string) => {
-      const { contents } = await createFlowBuilderPluginAndGetContents({
-        flowBuilderOptions: { flow: basicFlow },
-        requestArgs: {
-          input: { data: inputData, type: INPUT.TEXT },
-        },
-      })
+      const { contents, request, flowBuilderPluginPost } =
+        await createFlowBuilderPluginAndGetContents({
+          flowBuilderOptions: { flow: basicFlow },
+          requestArgs: {
+            input: { data: inputData, type: INPUT.TEXT },
+          },
+        })
 
       expect((contents[0] as FlowText).text).toBe('Welcome message')
+      expect(request.input.nluResolution).toEqual({
+        type: 'keyword',
+        matchedValue: inputData,
```

Comment:
The `matchedValue` should be the keyword not the input of the user, no?

Reply:
`matchingValue` maybe makes more sense

Reply:
the matchedValue will be the keyword not the entire user input. But in this test the user input is only the keyword